runtime/trappist/src/lib.rs

```diff
@@ -683,9 +666,6 @@ construct_runtime!(
 		// Additional pallets
 		Dex: pallet_dex::{Pallet, Call, Storage, Event<T>} = 110,
 		AssetRegistry: pallet_asset_registry::{Pallet, Call, Storage, Event<T>} = 111,
-
-		// Chess
-		Chess: pallet_chess::{Pallet, Call, Storage, Event<T>} = 120,
```

Comment:
I will test this with the rococo relay chain at release 0.9.39 and verify this.

Reply:
I'm still hesitant about this one, I tried using release 0.9.39 and faced the same issue with the validator of the relay chain and Trappist can't produce blocks

Reply:
i feel in the same way as you do with the pallet_chess and my choice is to remove this pallet from trappist since it has demonstrated to be problematic from the wasm perspective. But this is to be discussed in a different issue / PR